src/main/java/org/testng/TestRunner.java

```diff
@@ -1613,11 +1584,15 @@ public Injector getInjector(List<Module> moduleInstances) {
   @Override
   public Injector getInjector(IClass iClass) {
     Annotation annotation = AnnotationHelper.findAnnotationSuperClasses(Guice.class, iClass.getRealClass());
-    if (annotation == null) return null;
```

Comment:
FYI I'm fine with `if/else` clauses on the same line if they only take one line.